quality/docs/qe_segmentation-att-satser.md

```diff
@@ -0,0 +1,44 @@
+# Quality estimation of core components of the Motion: att-satser
+
+## Summary
+
+We want to know the general quality of the dimensions of the motions corpus. The first step in that is to quality control the segmentation of documents
+
+
+## What is the problem
+
+We want to estimate the quality of the corpus by core dimensions, namely:
+
+1. The Block containing "att-satser"
+
+
+## Estimation procedure
+
+A stratified random sample will be generated that will be manually annotated by experts at the library. Then these annotations will be compared with the data to estimate differences compared with the data. This annotated file will stored under `quality/data/att-satser.csv`.
+
+
+### Sampling plan
+
+A random sample of three motions per chamber and per year. 
+
+The final estimate will based on a stratified random sample.
+
+
+### Annotation guidelines
+
+The annotator read the original motion document and write down the following in a CSV file, as one column of motion ID and additional columns for annotated data:
+
+	1. att_satser
```

Comment:
Maybe this needs to be defined? I would just add "[needs to be defined]" after to make this clear so we don't loose this.

Reply:
Or maybe the definition is below?